requirements/dev.txt

```diff
@@ -1,5 +1,4 @@
 -r release.txt
-pretty-errors==1.2.19
-check-manifest==0.46
-pytest==6.2.3
-faker==8.12.1
+check-manifest==0.49
+pytest==7.3.1
```

Comment:
I thought dependabot handled updating these versions. Thank you for taking care of it!

Reply:
Could it be possible that dependabot is configured to only run against `requirements/release.txt`?